docs/src/languages/php/xdebug.md

```diff
@@ -6,59 +6,71 @@ sidebarTitle: "Xdebug"
 
 [Xdebug](https://xdebug.org/) is a real-time debugger extension for PHP.
 While usually used for local development, it can also be helpful for debugging aberrant behavior on the server.
-It is available on Platform.sh Grid instances running PHP 7.2 and higher.
+It's available on Platform.sh Grid instances running PHP 7.2 and higher.
 
 As configured on Platform.sh, it avoids any runtime overhead for non-debug requests, even in production, and only allows connections via SSH tunnels to avoid any security issues.
 
 Note Xdebug runs only on your app containers.
 So you can't use it for [worker containers](../../create-apps/workers.md).
 
-## Setting up Xdebug
+## Before you begin
 
-Xdebug isn't enabled the same way as other extensions, as it should not be active on most requests.
-Xdebug has a substantial impact on performance and should not be run in a production process.
-Instead, Platform.sh runs a second PHP-FPM process with Xdebug that is used only for debug requests, leaving the normal process unaffected.
+You need to have:
 
-Enable Xdebug by adding the following configuration to the application's `.platform.app.yaml` file:
+- PHP 7.2+ on your project
+- The Platform.sh [CLI](../../administration/cli/_index.md)
+- A Xdebug-compatible IDE installed on your machine
 
-```yaml
+    For setup instructions, consult your IDE's Xdebug documentation.
+  Optionally, you can use [PHPStorm](https://www.jetbrains.com/phpstorm/), for which detailed instructions are provided.
```

Comment:
:clipboard: 
```suggestion
  For setup instructions, consult your IDE's Xdebug documentation.
  Optionally, you can use [PHPStorm](https://www.jetbrains.com/phpstorm/), for which detailed instructions are provided.
```

Reply:
:hammer_and_wrench: I don't think we need to document third-party tools. Especially as I bet the screenshots are rather out of date and potentially more confusing than helpful. I wouldn't get into any of that here.
```suggestion
  For setup instructions, consult your IDE's Xdebug documentation, such as that for [PHPStorm](https://www.jetbrains.com/help/phpstorm/configuring-xdebug.html).
```

Or really, could just link to examples in step 3.